openvdb/openvdb/tools/LevelSetFilter.h

```diff
@@ -95,6 +95,19 @@ class LevelSetFilter : public LevelSetTracker<GridT, InterruptT>
         Filter f(this, mask); f.meanCurvature();
     }
 
+    /// @brief One iteration of filleting on the level set.
```

Comment:
Would be great to add @details that provides a reference to where the technique comes from. If I recall correctly, it's from one of Sethian/Osher's book.